crypto/Cargo.toml

```diff
@@ -40,6 +40,7 @@ rstest_reuse = "0.4"
 rcgen = { version = "0.9", features = ["x509-parser"] }
 async-std = { version = "1.12", features = ["attributes"] }
 futures-util = { version = "0.3", features = ["std", "alloc"] }
+ed25519-dalek = "1.0"
```

Comment:
Maybe we can find a way to call into the MLS Crypto provider for this? It already depends on `ed25519-dalek`